docs/getting-started/tutorials/local_setup.md

```diff
@@ -51,8 +51,8 @@ Java HotSpot(TM) 64-Bit Server VM (build 25.111-b14, mixed mode)
 
 <div data-lang="Download and Unpack" markdown="1">
 1. Download a binary from the [downloads page](http://flink.apache.org/downloads.html). You can pick
```

Comment:
Offtopic, but can be improved to use `{{ site.download_url }}` instead of explicit downloads url.